doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -174,13 +174,13 @@ your new OSD before it can begin receiving data. You may use
 ``service ceph`` from your admin host or start the OSD from its host
 machine.
 
-For Debian/Ubuntu use Upstart. ::
+For Ubuntu Trusty use Upstart. ::
 
 	sudo start ceph-osd id={osd-num}
 
-For CentOS/RHEL, use sysvinit. ::
```

Comment:
I think we should keep old as well as new unless user is using the writing 'branch' doc or it would be confusing with various versions in use.